client_openapi.yaml

```diff
@@ -3466,6 +3466,13 @@ components:
           type: integer
         owned_by:
           type: string
+        permission:
+          type: array
+          items: {}
+        root:
+          type: string
+        parent:
+          type: string
```

Comment:
Shall we try setting up the `additionalProperties: true` instead, as follows? That should transform the counterpart ballerina record into an open record, which will allow having additional properties like the above

```yaml
Model:
   title: Model
   additionalProperties: true
   properties:
       ...
```